README.md

```diff
@@ -112,41 +122,55 @@ const distinctShareReplay = <T>(compare?: Object.is) => (
 ```
 
 The enhanced observables returned from `connectObservable` and `connectFactoryObservable` 
-have been enhanced like this:
+have been enhanced with this operator.
+
+### SUSPENSE
 
 ```ts
-const shared$ = concat(original$, NEVER).pipe(distinctShareReplay())
+const story$ = selectedStoryId$.pipe(
+  switchMap(id => concat(
+    SUSPENSE,
+    getStory$(id)
+  ))
+)
 ```
 
-### SUSPENSE
-
-This is a special symbol that can be emitted from our observables to let the react-hook
+This is a special symbol that can be emitted from our observables to let the react hook
 know that there is a value on its way, and that we want to leverage React Suspense API
 while we are waiting for that value.
 
 ### suspend
 
-A RxJS creation operator that performs a `startWith(SUSPENSE)` on the source observable.
-
 ```ts
-const suspend = <T>(source$: ObservableInput<T>) =>
-  from(source$).pipe(startWith(SUSPENSE))
+const story$ = selectedStoryId$.pipe(
+  switchMap(id => suspend(getStory$(id))
+)
 ```
 
+A RxJS creation operator that prepends a `SUSPENSE` on the source observable.
+
 ### suspended
 
+```ts
+const story$ = selectedStoryId$.pipe(
+  switchMap(id => getStory$(id).pipe(
+    suspended()
+  ))
+)
+```
+
 The pipeable version of `suspend`
 
 ### switchMapSuspended
 
-Like `switchMap` but applying a `startWith(SUSPENSE)` to the inner observable.
-
 ```ts
-const switchMapSuspended = <Input, Output>(
-  fn: (input: Input) => ObservableInput<Output>,
-) => (src$: Observable<Input>) => src$.pipe(switchMap(x => suspend(fn(x))))
+const story$ = selectedStoryId$.pipe(
+  switchMapSuspended(id => getStory$(id))
+)
```

Comment:
I love how you have been building it up so that the user can appreciate the value of this "sugar" operator... That's precisely why I think that for this example it would be worth it to use a point-free style example:

```ts
const story$ = selectedStoryId$.pipe(
  switchMapSuspended(getStory$)
)
```

In other examples I have avoided using point-free style declarations (like in `connectFactoryObservable`) because I wanted to make sure that the user understood the parameters that the function was receiving. However, in this particular case and taking into account that the docs have been building up this momentum... I think that we must :smile: